### PR TITLE
Build/336 update peer data

### PR DESCRIPTION
**Pull request by AlexAxthelm**

### CODE REVIEW

File: .github/workflows/build-push-private.yml [hunk 1]
```diff
@@ -351,6 +351,9 @@ jobs:
           mkdir -p $TEST_DIR/user_dir
           cp -R working_dir $TEST_DIR
 
+          # Remove FAKE string (used when testing priave images)
```
Comment (cjyetman) [inline .github/workflows/build-push-private.yml:354]:
```suggestion
          # Remove FAKE string (used when testing private images)
```

File: build/config/rmi_pacta_2023q4_FAKEpa2024ch.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "data_share_path": "2023Q4_20240424T120055Z",
```
Comment (jdhoffa) [inline build/config/rmi_pacta_2023q4_FAKEpa2024ch.json:2]:
```suggestion
  "data_share_path": "2023Q4_20240709T111731Z",
```

Relates to https://github.com/RMI-PACTA/pacta.data.preparation/pull/32

**Review by jdhoffa — APPROVED** [review]
lgtm

**Review by jdhoffa — APPROVED** [review]
lgtm

**Pull request merged**